Assets/Tests/EditMode/Inventory/StorageTests.cs

```diff
@@ -0,0 +1,132 @@
+using UnityEngine;
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine.TestTools;
+using Moq;
+using TMPro;
+
+namespace Tests
+{
+    public class StorageTests
+    {
+        private Storage storage;
+
+        [SetUp]
```

Comment:
Not an issue, but just for the record, you aren't required to have a SetUp method in your test files. Just in case you thought it was required.

Reply:
Removed setup method in comit f47a644